src/mscorlib/corefx/System/Globalization/CalendarData.Unix.cs

```diff
@@ -38,10 +67,53 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
 
         private static bool SystemSupportsTaiwaneseCalendar()
         {
-            // TODO: Implement this.
-            return false;
+            return true;
         }
 
         // PAL Layer ends here
+
+        private static bool GetCalendarInfo(string localeName, CalendarId calendarId, CalenderDataType dataType, out string calendarString)
+        {
+            calendarString = null;
+
+            const int stringSize = 80;
+            StringBuilder stringBuilder = StringBuilderCache.Acquire(stringSize);
+
+            bool result = Interop.GlobalizationInterop.GetCalendarInfo(localeName, calendarId, CalenderDataType.MonthDay, stringBuilder, stringSize);
```

Comment:
Did you mean to pass the formal parameter "dataType" to this call instead of the fixed MonthDay value?  If not, do we need the datatype parameter at all?